<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleInfoWidgetLayout</class>
 <widget class="QWidget" name="SampleInfoWidgetLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>151</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SampleInfo</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sample</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <layout class="QHBoxLayout" name="hbox">
          <item>
           <layout class="QVBoxLayout" name="label_vbox">
            <item>
             <widget class="QLabel" name="name_label">
              <property name="text">
               <string>Name:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="data_matrix_label">
              <property name="text">
               <string>Data matrix:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="holder_length_label">
              <property name="text">
               <string>Holder length:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="basket_location_label">
              <property name="text">
               <string>Basket location:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sample_location_label">
              <property name="text">
               <string>Sample location:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="value_vbox">
            <item>
             <widget class="QLabel" name="name_value_label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>32767</height>
               </size>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="data_matrix_value_label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>32767</height>
               </size>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="holder_length_value_label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>32767</height>
               </size>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="basket_location_value_label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>32767</height>
               </size>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sample_location_value_label">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>32767</height>
               </size>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
